runtime/doc/autocmd.txt

```diff
@@ -1011,6 +1011,9 @@ WinLeave			Before leaving a window.  If the window to be
 WinNew				When a new window was created.  Not done for
 				the first window, when Vim has just started.
 				Before WinEnter.
+							*WinScrolled*
+WinScrolled			After scrolling the viewport.
```

Comment:
Maybe we should mention that this only applies to current window. Otherwise early plugin tester are going to be very confused :)

Reply:
:+1: